po/es.po

```diff
@@ -200,6 +197,10 @@ msgid ""
 "domains like i❤tacos.ws will show up properly in the URL entry, while "
 "phishing domains should still show up like xn--80ak6aa92e.com"
 msgstr ""
+"🎉️ ¡Soporte de emoticonos (y caracteres internacionales) para dominios! "
```

Comment:
“Soporte” is not the right way to translate “support”. Use “compatibilidad” here instead.